lockup/README.md

```diff
@@ -1,43 +1,64 @@
 # Lockup / Vesting contract
 
-# PLEASE WAIT UNTIL [ISSUE 141](https://github.com/near/core-contracts/issues/141) IS RESOLVED BEFORE DEPLOYING THE CONTRACT
-
 ## Overview
 
-This contract acts as an escrow that locks and holds an owner's tokens for a lockup period. A lockup period starts
-from the moment transfers are enabled by voting and lasts for the specified duration.
+This contract acts as an escrow that locks and holds an owner's tokens for a lockup period. 
+The contract contains of lockup and vesting processes that go simultaneously.
+
+A lockup period starts from the specified timestamp (or, if not specified, from moment when transfers are enabled by voting) 
+and lasts for the specified duration.
```

Comment:
As suggested by @matklad, we start to use one sentence per line https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
@bowenwang1996 do we have some style.md somewhere where we could dump this info? A-la https://github.com/rust-analyzer/rust-analyzer/blob/d4a51241ceedae25f15bb8d202493554ae0af810/docs/dev/style.md#documentation ?